swim/state_transitions_test.go

```diff
@@ -102,19 +108,44 @@ func (s *StateTransitionsSuite) TestSuspectDisabled() {
 }
 
 func (s *StateTransitionsSuite) TestSuspectBecomesFaulty() {
-	s.stateTransitions.timeouts.Suspect = time.Millisecond
-
-	s.m.MakeAlive(s.suspect.Address, s.suspect.Incarnation)
+	s.m.MakeSuspect(s.suspect.Address, s.suspect.Incarnation)
 	member, _ := s.m.Member(s.suspect.Address)
 	s.Require().NotNil(member, "expected cannot be nil")
 
 	s.stateTransitions.ScheduleSuspectToFaulty(*member)
-	s.NotNil(s.stateTransitions.timer(member.Address), "expected suspicion timer to be set")
+	s.NotNil(s.stateTransitions.timer(member.Address), "expected state transtition timer to be set")
 
-	time.Sleep(5 * time.Millisecond)
+	s.clock.Add(5*time.Second + 1)
```

Comment:
I prefer to not use the automatic conversion from 1 to Nanosecond (or whatever duration it is)